SPECS/openssl/openssl.spec

```diff
@@ -256,9 +252,15 @@ rm -f $RPM_BUILD_ROOT%{_sysconfdir}/pki/tls/ct_log_list.cnf.dist
 %postun libs -p /sbin/ldconfig
 
 %clean
-rm -rf $RPM_BUILD_ROOT
+rm -rf %{buildroot}
+
+
+
 
```

Comment:
Nit: I think we've had plenty of empty lines here already, no need for another one.;) Might be good feedback for Tom, to update the linter.